tools/hack/create-cluster.sh

```diff
@@ -12,6 +12,8 @@ NUM_WORKERS=${NUM_WORKERS:-""}
 KIND_CFG=$(cat <<-EOM
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: dual
```

Comment:
can you make this paramable? like
```
IP_FAMILY=${IP_FAMILY: ipv4_only}
```

Reply:
That's a good idea, and I've modified it.